ovs/lib/storagedriver.py

```diff
@@ -84,26 +84,35 @@ def update_status(storagedriver_id):
         if pmachine.mgmtcenter:
             # Update status
             pmachine.invalidate_dynamics(['host_status'])
+            host_status = pmachine.host_status
         else:
             # No management Center, cannot update status via api
             StorageDriverController._logger.info('Updating status of pmachine {0} using SSHClient'.format(pmachine.name))
             host_status = 'RUNNING'
             try:
                 client = SSHClient(storagerouter, username='root')
-                configuration_dir = EtcdConfiguration.get('/ovs/framework/paths|cfgdir')
                 StorageDriverController._logger.info('SSHClient connected successfully to {0} at {1}'.format(pmachine.name, client.ip))
-                with remote(client.ip, [LocalStorageRouterClient]) as rem:
-                    lsrc = rem.LocalStorageRouterClient('{0}/storagedriver/storagedriver/{1}.json'.format(configuration_dir,
-                                                                                                          storagedriver.vpool.name))
-                    lsrc.server_revision()
-                StorageDriverController._logger.info('LocalStorageRouterClient connected successfully to {0} at {1}'.format(pmachine.name, client.ip))
-            except Exception as ex:
-                StorageDriverController._logger.error('Connectivity check failed, assuming host {0} is halted. {1}'.format(pmachine.name, ex))
+            except UnableToConnectException as ex:
+                StorageDriverController._logger.error('SSHClient connectivity check failed, assuming host {0} is halted. {1}'.format(pmachine.name, ex))
                 host_status = 'HALTED'
-            if host_status != 'RUNNING':
-                # Host is stopped
-                storagedriver_client = StorageDriverClient.load(storagedriver.vpool)
-                storagedriver_client.mark_node_offline(str(storagedriver.storagedriver_id))
+            else:
+                try:
+                    with remote(client.ip, [LocalStorageRouterClient]) as rem:
+                        path = 'etcd://127.0.0.1:2379/ovs/vpools/{0}/hosts/{1}/config'.format(storagedriver.vpool.guid,
```

Comment:
Can you create a ```StorageDriverConfiguration``` object (that can load the config based on vpool etc), and then use ```.remote_path``` property. If we have to move the config location around, we only have to update that one.